config/application.rb

```diff
@@ -15,6 +15,8 @@ class Application < Rails::Application
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
 
+    config.autoload_paths << Rails.root.join('lib')
```

Comment:
Don't do this as this is a a bad practice, files from lib must loaded explicitly, you can keep others in app/lib or app/services